server/api.go

```diff
@@ -489,9 +489,16 @@ func (p *Plugin) getUnreads(w http.ResponseWriter, r *http.Request, userID strin
 			continue
 		}
 
+		issueURL := n.GetSubject().GetURL()
+		issueNum := string(issueURL[len(issueURL)-1])
```

Comment:
This would only work in the case that the issue number is one digit. Otherwise you will need to work off of the last slash, using something like `strings.LastIndex(issueURL, "/")`